cmd/internal/database/postgres/postgres.go

```diff
@@ -164,5 +166,30 @@ func (db *Postgres) Probe(ctx context.Context) error {
 	if err != nil {
 		return fmt.Errorf("unable to ping postgres connection %w", err)
 	}
+
+	runsTimescaleDB, err := db.runningTimescaleDB(ctx, postgresConfigCmd)
+	if err == nil && runsTimescaleDB {
+		db.log.Infow("detected running timescaledb, running post-start hook to update timescaledb extension if necessary")
+
+		_, err = dbc.ExecContext(ctx, "ALTER EXTENSION timescaledb UPDATE;")
+		if err != nil {
+			return fmt.Errorf("unable to alter extension: %w", err)
+		}
```

Comment:
@majst01 Can we somehow iterate through the databases that use the extension and update them? I think it could be dangerous to just update on `dbname=postgres` and `dbname=template1`.

Reply:
It is possible to list the database and the extensions from sql, so in theory this could be done and should be done.

Can assist of course

Reply:
:pray: